quotes.json

```
@@ -153,6 +153,8 @@
   },
   {
     "author": "Albert Einstein",
+    "source": "https://philosiblog.com/2011/04/09/if-you-cant-explain-it-simply/",
```

```suggestion
    "source": "https://en.wikiquote.org/wiki/Albert_Einstein",
```

https://philosiblog.com seems like a reasonably reliable website. 
Using Wikiquote as a source as it's guaranteed to be available/unbiased.